### PR TITLE
Update change log

### PR DESCRIPTION
**Pull request by pljones**
**Short description of changes**

Update ChangeLog for 3.11.0

CHANGELOG: SKIP

**Context**

Update ChangeLog for 3.11.0

**Does this change need documentation? What needs to be documented and how?**

No.

**Status of this Pull Request**

This is the pre-beta release status for review.  I'm not familiar with how best to merge the entries -- I've just tried sorting them into groups at this stage.

**What is missing until this pull request can be merged?**

Review, comments and feedback.

## Checklist

-  [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
-  [x] I tested my code and it does what I want
-  [x] My code follows the [style guide](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md#source-code-consistency) 
-  [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
-  [x] I've filled all the content above

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,4 +1,138 @@
...
+- Client: Added "delete server" button to connect dialog (#3159).
+  (contributed by @pljones)
+
+- Client: Added "Delete Entry" button to Advanced Settings, Custom Directories (#3260).
+  (contributed by @AdamGLIN)
+
+- Client: Fixed the display of delete buttons on Mac (#3305).
+  (contributed by @softins)
+
+- Client: Bugfix: Removed channel number from saved fader tag when --ctrlmidich is in use. In this release --cleanuplegacyfadersettings can be used on any saved fader settings that have been corrupted to attempt recovery (#3116).
```
Comment (pljones) [inline ChangeLog:18]: This entry was tagged `CHANGELOG: SKIP` -- not sure why it's here.

File: ChangeLog [hunk 2]
```diff
@@ -1,4 +1,138 @@
...
+  (contributed by @app/github-actions)
+
+- Build: Updated bundled Qt6 to version 6.6.1 (#3204).
+  (contributed by @app/github-actions)
+
+- Build: Updated bundled Qt6 to version 6.6.3 (#3236).
+  (contributed by @app/github-actions)
+
+- Build: Updated Windows Installer base (NSIS) to version 3.10 (#3254).
+  (contributed by @app/github-actions)
```
Comment (pljones) [inline ChangeLog:130]: Missing blank line.

File: ChangeLog (hunk 1 above)
Comment (pljones) [reply ChangeLog:18]: There was a quoted entry with the " C H A N G L O G " text in.

File: ChangeLog [hunk 3]
```diff
@@ -1,5 +1,112 @@
...
+
+- Translations updated from Hosted Weblate (#3246).
+  (contributed by @weblate)
+
+- Documentation: Improved link for Server CLI options (#3303).
+  (contributed by @gilgongo)
+
+- Documentation: updated TRANSLATING.md for current build process (#3312).
+  (contributed by @softins)
+- Build: adds macOS signed pkg build automation (#2624).
```
Comment (ann0see) [inline ChangeLog:44]:
```suggestion

- Build: adds macOS signed pkg build automation (#2624).
```

File: ChangeLog [hunk 4]
```diff
@@ -1,5 +1,113 @@
...
+- Build: Bump actions/checkout from 3 to 4 (#3168).
+  (contributed by @app/dependabot)
+
+- Build: Bump maxim-lobanov/setup-xcode from 1.5.1 to 1.6.0 (#3181).
+  (contributed by @app/dependabot)
+
+- Build: Updated bundled Qt6 to version 6.6.0 (#3182).
+  (contributed by @app/github-actions)
+
+- Build: Ensure apt version >=2.4 to prove that the system is Debian based (using apt and dpkg) and can validate the repo (#3199).
```
Comment (pljones) [inline ChangeLog:57]: Hm.  I edited this one in the PR, too...  Not sure why it's still got the old text.  Probably a " C H A N G E L O G " tag in a comment I missed.

**Review by ann0see — COMMENTED** [review]
Let's put this in for now. We may still want to rewrite it to be more user centric before the final release

**Pull request merged**